services/api/db/admin.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib import admin
+from db.models import User
+
+# Now register the new UserModelAdmin...
+admin.site.register(User)
```

Comment:
```suggestion
admin.site.register(User)
```
please handle it appropriately

Reply:
deleted it